packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -313,6 +319,91 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
   // stylelint-enable plugin/selector-bem-pattern
 }
 
+.mdc-textfield--with-leading-icon .mdc-textfield__leading-icon,
+.mdc-textfield--with-trailing-icon .mdc-textfield__trailing-icon {
+  position: absolute;
+  bottom: 16px;
+  cursor: pointer;
+
+  @include mdc-theme-dark(".mdc-textfield") {
+    @include mdc-theme-prop(color, text-secondary-on-dark);
+  }
+}
+
+.mdc-textfield--with-leading-icon {
+  .mdc-textfield__input {
+    @include mdc-rtl-reflexive-property(padding, $mdc-textfield-icon-padding, $mdc-textfield-icon-position);
+  }
+
+  .mdc-textfield__leading-icon {
+    @include mdc-rtl-reflexive-position(left, $mdc-textfield-icon-position);
+  }
+
+  .mdc-textfield__label {
+    @include mdc-rtl-reflexive-position(left, $mdc-textfield-icon-padding);
+  }
+}
+
+.mdc-textfield--with-trailing-icon {
+  .mdc-textfield__input {
+    @include mdc-rtl-reflexive-property(padding, $mdc-textfield-icon-position, $mdc-textfield-icon-padding);
+  }
+
+  .mdc-textfield__trailing-icon {
+    @include mdc-rtl-reflexive-position(right, $mdc-textfield-icon-position);
+  }
+}
+
+.mdc-textfield__leading-icon:not([tabindex]),
+.mdc-textfield__leading-icon[tabindex="-1"] {
+  pointer-events: none;
+  cursor: default;
+}
+
+.mdc-textfield__trailing-icon:not([tabindex]),
```

Comment:
Why not combine these 2 selectors with the previous 2? Can easily avoid repeating the 2 rules inside.